ArcGISRuntimeSDKQt_QMLSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,374 @@
+                        rangeValuesSpinBox.to = minMax[1];
+
+                        // Explicitly set the buffer size to the rangeValuesSpinBox value in case onValueChanged doesn't trigger
+                        newFeature.attributes.replaceAttribute("BufferSize", rangeValuesSpinBox.value);
+                        saveButton.enabled = validateContingentValues();
+                    }
+                }
+            }
+
+            Text {
+                text: "Buffer Size"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            Text {
+                text: rangeValuesSpinBox.from + " to " + rangeValuesSpinBox.to;
+            }
+
+            SpinBox {
+                id: rangeValuesSpinBox
+                editable: true
+                from: 0;
+                to: 0;
+                stepSize: 10
+                value: 0;
+
+                enabled: protectionComboBox.currentText !== ""
+
+                onValueChanged: {
+                    if (protectionComboBox.currentValue === "")
+                        return;
+
+                    newFeature.attributes.replaceAttribute("BufferSize", rangeValuesSpinBox.value);
+
+                    // Validate that all contingencies are valid, if so, enable the save button
+                    saveButton.enabled = validateContingentValues();
+                }
+            }
+
+            Button {
+                id: saveButton
+                Text {
+                    anchors.fill: parent
+                    horizontalAlignment: Text.AlignHCenter
+                    verticalAlignment: Text.AlignVCenter
+                    text: "Save"
+                }
+
+                enabled: false
+
+                onClicked: {
+                    const valid = validateContingentValues();
+                    if (validateContingentValues()) {
+                        createNewNest();
+                        attributePrompt.visible = false;
+                    }
+                }
+            }
+
+            Button {
+                id: discardButton
+                Text {
+                    anchors.fill: parent
+                    horizontalAlignment: Text.AlignHCenter
+                    verticalAlignment: Text.AlignVCenter
+                    text: "Discard"
+                }
+                onClicked: {
+                    attributePrompt.visible = false
+                }
+            }
+        }
+
+        onVisibleChanged: {
+            if (!visible)
+                return;
+
+            // Reset attribute panel values when the panel opens
+            statusComboBox.currentIndex = 0;
+            protectionComboBox.currentIndex = 0;
+            rangeValuesSpinBox.from = 0;
+            rangeValuesSpinBox.to = 0;
+        }
+    }
+
+    function getContingentValues(field, fieldGroup) {
+        const contingentValuesResult = birdNestsTable.contingentValues(newFeature, field);
```

Comment:
When I select an option from the Status dropdown menu, I get an error regarding contingentValues here, complaining that it is not a function.

> qrc:/Samples/EditData/ContingentValues/ContingentValues.qml:298: TypeError: Property 'contingentValues' of object QmlGeodatabaseFeatureTable(0x28a546d8230) is not a function

Reply:
Again, this is probably a result of the definition not loading properly